be/src/vec/common/hash_table/hash_table.h

```diff
@@ -612,6 +555,62 @@ class HashTable : private boost::noncopyable,
         return *this;
     }
 
+        /// Increase the size of the buffer.
+    void resize(size_t for_num_elems = 0, size_t for_buf_size = 0) {
+        SCOPED_RAW_TIMER(&_resize_timer_ns);
```

Comment:
why move this function? it seems no changes

Reply:
HashTable::resize  moved from protected to public, so that it could be invoked by ProcessHashTableBuild in vhash_join_node.cpp

Reply:
the test result： 1419 ms -> 1293 ms
tested 3 times, the average BuildTableInsertTime decreased by 8.8% (=  (1419-1293) / 1419 )